src/component/menu/drawer.jsx

```diff
@@ -17,8 +17,8 @@ const filterByFlags = flags => r => {
     return true;
 };
 
-function getIcon(name) {
-    if (name === 'c_github') {
+function getIcon(IconComponent) {
```

Comment:
The "library_books" icon is actually coming from our API, which will override the default setting you are setting in the reducer further down and render `<library_books>` into the DOM causing a warning. We need to add a check here for "library_books" like we are doing with the github icon.